website/package.json

```diff
@@ -13,11 +13,12 @@
     "@hashicorp/react-command-line-terminal": "^2.0.4",
     "@hashicorp/react-consent-manager": "^7.0.1",
     "@hashicorp/react-content": "^8.0.2",
-    "@hashicorp/react-docs-page": "^14.2.4",
+    "@hashicorp/react-docs-page": "^14.4.2",
     "@hashicorp/react-hashi-stack-menu": "^2.0.7",
     "@hashicorp/react-head": "^3.1.2",
     "@hashicorp/react-image": "^4.0.3",
     "@hashicorp/react-product-downloads-page": "^2.5.2",
+    "@hashicorp/react-search": "^6.1.1",
```

Comment:
Waaait a second we dont need to add this right? It should be a dependency of docs-page